app/src/main/java/com/incwelltechnology/lms/data/network/LmsApi.kt

```diff
@@ -0,0 +1,27 @@
+package com.incwelltechnology.lms.data.network
+
+import com.incwelltechnology.lms.data.model.LoginRequest
+import com.incwelltechnology.lms.data.model.LoginResponse
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+interface LmsApi {
+
+    @POST("login")
+    fun userLogin(
+        @Body login: LoginRequest
+    ): Call<LoginResponse>
+
+    companion object{
```

Comment:
@iamsannyrai why did you make the companion object for accessing network request?

Reply:
Since we are using LmsApi interface for all our api calls, using companion object reduces the task of creating object each time (or makes object independent) and hence makes function defined there, directly accessible from class (or class name)